zerver/models/users.py

```diff
@@ -72,6 +72,7 @@ class UserBaseSettings(models.Model):
     display_emoji_reaction_users = models.BooleanField(default=True)
     twenty_four_hour_time = models.BooleanField(default=False)
     starred_message_counts = models.BooleanField(default=True)
+    automatically_detect_time_zone = models.BooleanField(default=True)
```

Comment:
Please start an "api design" conversation for what to call this. I suspect at the very least, this should have `web_` as a prefix in its name. Maybe `web_suggest_update_timezone` would be more precise?

Reply:
I've initiated a [conversation](https://chat.zulip.org/#narrow/stream/378-api-design/topic/Automatically.20detect.20user's.20timezone) there. I'll change it to `web_suggest_update_timezone` for the time being